### PR TITLE
new major release compatible with Mirantis galera

### PR DESCRIPTION
**Pull request by LotharKAtt**

### CODE REVIEW

File: galera/master.sls [hunk 1]
```diff
@@ -63,6 +71,7 @@ galera_bootstrap_script:
 
 {%- if salt['cmd.run']('test -e /var/lib/mysql/.galera_bootstrap; echo $?') != '0'  %}
 
+{#
```
Comment (fpytloun) [inline galera/master.sls]: Don't comment out, remove completely

File: galera/master.sls [hunk 2]
```diff
@@ -72,12 +81,13 @@ galera_bootstrap_temp_config:
...
     - pkg: galera_packages
     - file: galera_init_script
+#}
 
 galera_bootstrap_start_service:
   cmd.run:
   - name: /usr/local/sbin/galera_bootstrap.sh
   - require:
-    - file: galera_bootstrap_temp_config
+#    - file: galera_bootstrap_temp_config
```
Comment (fpytloun) [inline galera/master.sls]: Same here

File: galera/slave.sls [hunk 3]
```diff
@@ -63,21 +71,21 @@ galera_bootstrap_script:
...
-galera_bootstrap_temp_config:
-  file.managed:
-  - name: {{ slave.config }}
-  - source: salt://galera/files/my.cnf.bootstrap
-  - mode: 644
-  - template: jinja
-  - require: 
-    - pkg: galera_packages
-    - file: galera_init_script
+#galera_bootstrap_temp_config:
```
Comment (fpytloun) [inline galera/slave.sls]: And here

**Comment by pupapaik** [comment]
LGTM

**Pull request merged**